OwenConferenciasApp/screens/formsLogin/MainLoginLayout.js

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import {View, Text, Image, TouchableOpacity, StyleSheet} from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import SignIn from './SignInLayout';
+import SignUp from './SignUpLayout';
+import App from '../../App';
+
+function MainForm({navigation}) {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' , backgroundColor:'#F0F1F6'}}>
+      <View style={styles.mainContainer}>
+        <Text style={styles.title}> Welcome to BC👨‍💻 Agenda 📝</Text>
+        <View>
+          <Image
+            source={require('../../assets/logo-bc.png')}
+            style={styles.imgLogo}
+          />
+        </View>
+        <View style={styles.containerBtns}>
+          <View style={[styles.btnsLogin, styles.btnSignIn]}>
+            <TouchableOpacity onPress={() => navigation.navigate('SignIn')}>
+              <Text style={styles.textBtns}>Sign In</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={[styles.btnsLogin, styles.btnSignUp]}>
+            <TouchableOpacity onPress={() => navigation.navigate('SignUp')}>
+              <Text style={styles.textBtns}>Sign Up</Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+      </View>
+    </View>
+  );
+}
+
+const Stack = createStackNavigator();
+
+export default function goSignIn() {
```

Comment:
You need to define navigation structure on a specific component, something like `navigation.js`, here you are mixing it with another compoentn